src/templates/home-page.js

```diff
@@ -74,14 +74,13 @@ const BLMSolidarityModal = () => {
           </p>
         </main>
         <p style={{marginBottom:"1rem"}}>
-            Please consider donating to <a href="https://bailfunds.github.io/">your local bail fund</a>,&nbsp;
-            <a href="https://codecooperative.org/">Code Cooperative</a>,&nbsp;
-            <a href="https://emergencyreleasefund.com/">Emergency Release Fund</a>,&nbsp;
-            <a href="https://spny.wedid.it/campaigns/7636/contribute">Survived & Punished Mutual Aid Fund</a>,&nbsp;
-            <a href="https://www.gofundme.com/f/9v4q2-justice-for-breonna-taylor">Justice for Breonna Taylor</a>,&nbsp;
-            <a href="https://www.gofundme.com/f/georgefloyd">George Floyd Memorial Fund</a>,&nbsp;
-            and
-            <a href="https://m4bl.org/">Movement for Black Lives</a>. 
+            Please consider donating to <a style={{color:"#A15FFB"}} href="https://bailfunds.github.io/">your local bail fund</a>,&nbsp;
```

Comment:
Regarding the google rating. I checked for that as well. Those are the purples, now even worse with the transparent background on top. Would be cool to redesign the purple, even though I really like the shade. 

I tried to be consistent with the style, I think it would be way better to have classes instead of inline. Would be good to clean that up.

Reply:
Sounds good. I'll open a few new issues and add them to the Website revision project!